src/bindeps_conda.jl

```diff
@@ -1,57 +1,46 @@
 # This file contains the necessary ingredients to create a PackageManager for BinDeps
 using BinDeps
 
-type Manager <: BinDeps.PackageManager
+type ManagerType{T} <: BinDeps.PackageManager
     packages::Vector{Compat.ASCIIString}
```

Comment:
For function arguments, you want to use `AbstractString` so that any string type is accepted.  For `type` fields, you want to use a concrete type, and `Compat.UTF8String` (same as `String` in Julia 0.5) is the right choice because it can hold any Unicode string data.